src/contracts/core/StrategyManager.sol

```diff
@@ -62,8 +64,9 @@ contract StrategyManager is
     constructor(
         IDelegationManager _delegation,
         IEigenPodManager _eigenPodManager,
-        ISlasher _slasher
-    ) StrategyManagerStorage(_delegation, _eigenPodManager, _slasher) {
+        ISlasher _slasher,
+        IAVSDirectory _avsDirectory
```

Comment:
are we using the AVSD?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/679/commits/55f38717e6c1303ea66d48236b312be5c6f0f86e leftover from initial scaled shares impl